web/cypress/e2e/onboarding-as-existing-business.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { onOnboardingPageExistingBusiness } from "@businessnjgovnavigator/cypress/support/page_objects/onboardingPageNew";
+import { arrayOfSectors } from "@businessnjgovnavigator/shared/";
+
+describe("Onboarding for all industries when starting a business [feature] [all] [group4]", () => {
```

Comment:
dust - title needs to be updated to match the test

Reply:
updated